cms/views.py

```diff
@@ -49,7 +49,7 @@
 
 if DJANGO_2_2:
     from django.utils.http import (
-        is_safe_url as url_has_allowed_host_and_scheme,
+        url_has_allowed_host_and_scheme as url_has_allowed_host_and_scheme,
```

Comment:
If this check stays, we don't need the "as" because it's the same name.

Reply:
Nope, I think this is an automatic change. Only the else part should stay. `DJANGO_2_2` means version 2.2 or lower. It can go, since we drop Django 2.2 support.

Reply:
That's what I was hoping!

Reply:
I cleaned up this code and all if conditions referencing DJANGO_2_2, DJANGO_3_0 and DJANGO_3_1

Reply:
@vinitkumar Have you pushed? I can't see any commits since Saturday.

Reply:
Omg! I am stupid. I have been pushing on my fork and not upstream and expecting you to see the changes. Sorry!

Reply:
Thanks!